### PR TITLE
Adding arrows function to render function of dagre-d3

### PR DESCRIPTION
**Pull request by ryanmcnamara**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: dagre-d3/dagre-d3.d.ts [hunk 1]
```diff
@@ -23,6 +24,8 @@ declare namespace Dagre {
     }
 
     interface Render {
+        // see http://cpettitt.github.io/project/dagre-d3/latest/demo/user-defined.html for example usage
```
Comment (markwongsk) [inline dagre-d3/dagre-d3.d.ts]: I don't think this comment is necessary? If you feel strongly about documenting, linking to source might be more useful? https://github.com/cpettitt/dagre-d3/blob/master/lib/arrows.js

**Comment by markwongsk** [comment]
👍

**Comment by horiuchi** [comment]
@ryan10132 Thanks!
@markwongsk Thanks for review!

**Comment by ChristianBeilschmidt** [comment]
Are you really sure that this reference to JQuery is necessary and correct?

**Comment by markwongsk** [comment]
From what I see, the call to .append() indicates that it has an inherent
dependency on JQuery since otherwise it would be .appendChild().

- Mark

On 25 May 2016 at 01:57, Christian Beilschmidt <notifications@github.com>
wrote:

> Are you really sure that this reference to JQuery is necessary and correct?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9398#issuecomment-221513571>
>



-- 
--------------------------------
Mark Wong Siang Kai
CMU Computer Science Class of 2013
CMU Chemical Engineering Class of 2013

**Comment by ChristianBeilschmidt** [comment]
But I can't see a dependency to JQuery in [dagre-d3](https://github.com/cpettitt/dagre-d3/blob/master/package.json) nor in [d3](https://github.com/d3/d3/blob/master/package.json).

**Comment by ryan10132** [comment]
Ahh, @ChristianBeilschmidt you're correct. 
Looking at https://github.com/cpettitt/dagre-d3/blob/master/lib/create-edge-paths.js#L56 it's apparent that that parameter should be a d3 selection type

**Comment by ryan10132** [comment]
@ChristianBeilschmidt I think you might be right, looking at https://github.com/cpettitt/dagre-d3/blob/master/lib/create-edge-paths.js#L56

**Pull request merged**